### ISSUE TITLE
DXF Text is missplaced after export

### ISSUE THREAD

**Issue by Steinblock** [SOFAgh/CADability#129]
I have a simple DXF file with a text object (the yellow one)

![image](https://user-images.githubusercontent.com/6850324/210377997-03989579-d3ae-47e3-ae72-037d8079b151.png)

If I export the file as DXF the text is at the wrong position (zero) and has a wrong orientation.

![image](https://user-images.githubusercontent.com/6850324/210377826-7d405dd8-f09b-44d1-aef8-54230da4d7e9.png)

Files:  
[original.zip](https://github.com/SOFAgh/CADability/files/10337362/original.zip)  
[after_export.zip](https://github.com/SOFAgh/CADability/files/10337364/after_export.zip)

**Comment by Steinblock** [comment]
So the reason that happens is because in `ExportDxf.cs` the line `res.TransformBy(Matrix4(toText)); ` is commented out. 

```
        private netDxf.Entities.Text ExportText(GeoObject.Text text)
        {
            System.Drawing.FontStyle fs = System.Drawing.FontStyle.Regular;
            if (text.Bold) fs |= System.Drawing.FontStyle.Bold;
            if (text.Italic) fs |= System.Drawing.FontStyle.Italic;
            System.Drawing.Font font = new System.Drawing.Font(text.Font, 1000.0f, fs);
            netDxf.Entities.Text res = new netDxf.Entities.Text(text.TextString, Vector2.Zero, text.TextSize * 1000 / font.Height, new TextStyle(text.Font + ".ttf"));
            ModOp toText = ModOp.Fit(GeoPoint.Origin, new GeoVector[] { GeoVector.XAxis, GeoVector.YAxis, GeoVector.ZAxis }, text.Location, new GeoVector[] { text.LineDirection.Normalized, text.GlyphDirection.Normalized, text.LineDirection.Normalized ^ text.GlyphDirection.Normalized });
            // res.TransformBy(Matrix4(toText)); // easier than setting normal and rotation
            return res;
        }
```

https://github.com/SOFAgh/CADability/blob/82ffb3400f8971b8fb756c8e96a28f9019930eb0/CADability/ExportDxf.cs#L157

If I uncomment it again (and the used Matrix4 method), it works as intended. Is there a reason why this line has been commented out?

**Comment by SOFAgh** [comment]
We should probably uncomment this line. I tested with a few files, maybe you have some more files to test with.

**Comment by Steinblock** [comment]
I wrote a simple unit test plus fix and created a PR #130

Since even that simple example fails without the change I would assume this solution should be merged.
